sites/platform/src/domains/cdn/_index.md

```diff
@@ -70,6 +72,18 @@ To disable it, change your cache configuration for the routes behind a CDN to th
        # Disable the HTTP cache on this route. It's handled by the CDN instead.
        enabled: false
 ```
+<--->
+<!-- Upsun -->
+```yaml {configFile="routes"}
+routes:
+  "https://{default}/":
+     type: upstream
+     upstream: "app:http"
+     cache:
+         # Disable the HTTP cache on this route. It's handled by the CDN instead.
+         enabled: false
+```
+{{< /version/specific >}}
```

Comment:
@chadwcarlson Tier availability banner on l.91 (for PSH docs) doesn't render correctly when the {{< version/only "1" >}} shortcode is applied, couldn't fix it.